edgelet/build/linux/build.sh

```diff
@@ -6,6 +6,11 @@
 
 set -e
 
+###############################################################################
+# These are the packages this script will build.
+###############################################################################
+packages=(iotedge iotedged iotedge-diagnostics iotedge-proxy)
```

Comment:
We need `iotedge-proxy` for non-Kubernetes?

Reply:
iotedge-proxy is not build with the feature "runtime-kubernetes" - it's built with the default features.

I've gone back and forth on this.  It was easier to add to the list of packages, and where it is placed does not impact build time, but it not required for non-k8s work.  If other people have strong opinions on placement, I'll gladly move it.

Reply:
If it was meant for non-Kubernetes I would've liked to be on the code review for when it was merged. Just from a quick glance I see [it's not doing error-handling correctly.](https://github.com/Azure/iotedge/blob/master/edgelet/iotedge-proxy/src/error.rs#L127-L165)

Anyway, what features it's built with isn't the way we should differentiate. Was it planned to ship in `iotedge.deb` ? Then it's part of the regular build. If it wasn't, then it's not.

I'm fine with leaving it where it is, but in the future if you're planning on doing non-Kubernetes work, please involve the non-Kubernetes team in the code reviews.

Reply:
Added a bug to our backlog to get our error handling in sync with the rest of the project.